packages/dashboard/src/components/Wizard/Wizard.Base.tsx

```diff
@@ -19,33 +20,53 @@ export class WizardBase extends React.Component<IWizardProps, {}> {
       throw new Error('Wizard must have atleast one step.');
     }
 
-    const classNames = getClassNames(this.props.styles!, { theme: this.props.theme! });
-
     // if the step to render is already passed in, use that
     const wizardStepProps = this.props.stepToShow ? this.props.stepToShow : getStepToShow(this.props);
 
+    let parentStep: IWizardStepProps | undefined;
+    let isFirstSubStep: boolean = false;
+
+    if (wizardStepProps.isSubStep) {
+      parentStep = getParentStep(steps, wizardStepProps);
```

Comment:
I feel like we need to come up with a better way to solve for this. We map over the steps already in subway nav so maybe we can leverage that and pass some props through. Maybe we don't set a prop but maybe on a click we can pass the child index or something. This feels a bit dirty to me.

Reply:
I made a change to update getStepToShow method to also update the props we need.  We are anyway already calling this method and iterating through the steps to get the step to show.